histomicstk/segmentation/label/delete_border.py

```diff
@@ -41,6 +39,9 @@ def delete_border(im_label):
         if len(border_indices) == 0:
             return im_label
 
+        # Condense and delete the border regions from the image label
+        im_label_del = np.zeros_like(im_label)
         im_label_del = condense(delete(im_label, border_indices))
-
-    return im_label_del
+        return im_label_del
+    else:
```

Comment:
Because the previous block returns, the `else` is unnecessary.  The code flow might be easier to read if we did the converse and then more of the code would be outdented:
```
if not np.any(im_label):
    return im_label
im_border_mask = ...
...
```